tests/test_tree.rs

```diff
@@ -49,7 +50,7 @@ fn kv(i: usize) -> Vec<u8> {
 #[test]
 fn it_works() {
     println!("========== initial sets ==========");
-    let t = Tree::new();
+    let t = Arc::new(Tree::new());
```

Comment:
I think it's because I'm trying to open a file called `/dev/shm/...` which on linux machines actually just puts stuff into shared memory, but it's not present on OSX, so I'm going to use a trick described to me as "poor man's shared memory" where you create a file, then rm it when a process still has a reference open on it, which causes the pages to still exist in memory, but the writes never end up backed on disk.